solutions/search/hybrid-semantic-text.md

```diff
@@ -141,7 +149,7 @@ GET semantic-embeddings/_search
 4. The `semantic_text` field is used to perform the semantic search.
 
 
-After performing the hybrid search, the query will return the top 10 documents that match both semantic and lexical search criteria. The results include detailed information about each document:
+After performing the hybrid search, the query will return the top 10 documents that match both semantic and lexical search criteria. The results include detailed information about each document.
```

Comment:
It's not strictly necessary that results match both:

```suggestion
After performing the hybrid search, the query will return the combined top 10 documents for both semantic and lexical search criteria. The results include detailed information about each document.
```

Reply:
👍